.github/workflows/playwright.yml

```
@@ -0,0 +1,41 @@
+name: Playwright Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-playwright:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

This action is at v3 already, let's keep up with progress.

```suggestion
      - uses: actions/checkout@v3
```